src/vicsetup/libvicsetup/trace.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
```

Comment:
This should be integrated with SGX-LKL debug/trace output.

Reply:
Yes, I think so. I create an issue to track this: https://github.com/lsds/sgx-lkl/issues/607.